docs/example.py

```diff
@@ -12,11 +12,12 @@ def construct(self):
         self.play(ReplacementTransform(square, circle))
         self.wait()
         # Try typing the following lines
```

Comment:
Why were these hidden in the first place?

Reply:
test 4/15/2022

Reply:
1

Reply:
1